examples/python/gRPC/util/fb_helper.py

```diff
@@ -192,32 +297,72 @@ def getOrCreateProject(
     return projectUuid
 
 
-def createEmpty(builder):
-    """Create an empty flatbuffer"""
+def createEmpty(builder: Builder) -> bytearray:
+    """
+    Create an empty flatbuffer
+
+    Args:
+        builder: A flatbuffers Builder
+
+    Returns:
+        A bytearray representing the empty flatbuffer
+    """
     Empty.Start(builder)
     emptyMsg = Empty.End(builder)
     builder.Finish(emptyMsg)
     return builder.Output()
```

Comment:
I would return `bytes` as the type because the gRPC stubs require this type. 

```suggestion
def createEmpty(builder: Builder) -> bytes
    """
    Create an empty flatbuffer

    Args:
        builder: A flatbuffers Builder

    Returns:
        A bytearray representing the empty flatbuffer
    """
    Empty.Start(builder)
    emptyMsg = Empty.End(builder)
    builder.Finish(emptyMsg)
    return bytes(builder.Output())
```